### PR TITLE
OCPQE-26838 Implement 4.18 chain upgrade jobs -- part 2

### PR DESCRIPTION
**Pull request by JianLi-RH**
https://issues.redhat.com/browse/OCPQE-26838

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.6-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-4.18-upgrade-from-stable-4.16-azure-ipi-sno-etcd-encryption-arm-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.16-nutanix-ipi-zones-external-lb-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-stable-4.18-upgrade-from-stable-4.16-gcp-ipi-ovn-ipsec-arm-mixarch-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.6-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-4.18-upgrade-from-stable-4.16-azure-ipi-sno-etcd-encryption-arm-f28

**Comment by JianLi-RH** [comment]
4.16 to 4.18 upgrade passes: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-4.18-upgrade-from-stable-4.16-azure-ipi-sno-etcd-encryption-arm-f28/1858775882894675968

**Comment by JianLi-RH** [comment]
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-multi-stable-4.18-upgrade-from-stable-4.16-gcp-ipi-ovn-ipsec-arm-mixarch-f28/1858775969926483968 complains 
```
error: unable to parse image registry.build02.ci.openshift.org/ci-op-bsxlflyl/release@sha256:529fbe37da40215883391ff99e13afcbfee31649f838292076c670b7bc127898: cannot retrieve image configuration for manifest sha256:cd797d39609b75b65272107b5b5c305965ecb02ffa820fc21ad0568b836e521e in manifest list sha256:529fbe37da40215883391ff99e13afcbfee31649f838292076c670b7bc127898: unknown blob
```

**Comment by JianLi-RH** [comment]
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28/1858775720726106112
Timeout for a middle node

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.6-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-multi-stable-4.18-upgrade-from-stable-4.16-gcp-ipi-ovn-ipsec-arm-mixarch-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.1-aws-ipi-f28

**Comment by JianLi-RH** [comment]
4.16 to 4.18 stable upgrade passes:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-multi-stable-4.18-upgrade-from-stable-4.16-gcp-ipi-ovn-ipsec-arm-mixarch-f28/1859083511420424192

**Comment by JianLi-RH** [comment]
Upgrade from 4.7 to 4.18 failed with error `Entrypoint received interrupt: terminated`

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28/1859083432059998208

**Comment by JianLi-RH** [comment]
Upgrade from 4.1 to 4.18 failed with error Entrypoint received interrupt: terminated

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.1-aws-ipi-f28/1859083929651253248

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.6-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.1-aws-ipi-f28

**Comment by JianLi-RH** [comment]
4.1 to 4.18 upgrade failed after upgrading to 4.16： https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.1-aws-ipi-f28/1859219850900017152

**Comment by JianLi-RH** [comment]
Upgrade pass but job timeout: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.6-aws-ipi-f28/1859213448559202304

**Comment by JianLi-RH** [comment]
Upgrade pass but job timeout: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58957/rehearse-58957-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-4.18-upgrade-from-stable-4.7-aws-ipi-f28/1859219687276023808

**Comment by JianLi-RH** [comment]
hi @jhou1 The rebase removed LGTM and APPROVE, could you please add them again?

File: ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh [hunk 1]
```diff
@@ -292,9 +292,9 @@ cat << EOF > ~/fetch-kubeconfig.yml
     shell: >-
       oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
 
-  - name: Patching hostedcluster to disable all default sources
+  - name: Patching hostedcluster to disable all default sources"
```
Comment (liangxia) [inline ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:295]: Unexpected changes in this file

Comment (JianLi-RH) [reply ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh]: Good catch, it was changed by make update. I just revert it.

**Comment by JianLi-RH** [comment]
hi @jhou1 I just added below image to openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.16.yaml which is new introduced by https://github.com/openshift/release/pull/59029:
```
  vsphere-ci-python:
    name: vsphere-python
    namespace: ci
    tag: latest
```

**Pull request merged**